src/components/Settings.js

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable react-hooks/rules-of-hooks */
+
+import { useState } from 'react';
+import { useTheme } from '../App';
+import * as Api from '../Api';
+import SettingsIcon from "../images/settings-button.png";
+import '../styles/Settings.css';
+
+const Switch = ({ onChange, initialValue, name }) => {
+    const [value, setValue] = useState(initialValue || false);
+
+    return (
+        <div className={ useTheme("switch") }>
+            <input id={ `switch-input-${name.toLowerCase().replace(' ', '_')}` } checked={ value } type="checkbox"
+                   onChange={ e => {
+                       onChange(e.target.checked);
+                       setValue(e.target.checked);
+                   } } />
+            <label htmlFor={ `switch-input-${name.toLowerCase().replace(' ', '_')}` }>
```

Comment:
This is basically the same thing, you can use the variable you've already created.
```suggestion
            <label htmlFor={ `switch-input-${snakeCasedName}` }>
```